CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## stable-2.8.1
+
+This release fixes multicluster gateways support on EKS.
+
+* The `config.linkerd.io/proxy-destination-get-networks` annotation configures
+  the networks for which a proxy can discover metadata. This is an advanced
+  configuration option that has security implications.
```

Comment:
> that has security implications.

We should consider linking it to say the upgrade notes for more details. E.g., something like:

> See the upgrade notes for more details.

Reply:
Let's just omit this from the change log for stable, please. Users should not have to worry about this, except for extraordinary/advanced situations. There are no upgrade notes that are relevant to this.